.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: dart
+sudo: false
+
+dart:
+  - dev
+  - stable
+
+dart_task:
+  - test
+  - dartfmt
+  - dartanalyzer
```

Comment:
Only run these once each. See [the example file](https://github.com/dart-lang/oss_project_templates/blob/master/dart-github-dart-lang/.travis.yml).